src/components/MenuItem.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+function MenuItem({item, addOrder})  {
+
+function handleSubmit(event) {
+  event.preventDefault();
+
+  const min = 1;
+  const max = 100;
+  let keyId = Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
It would better to generate sequential ids on server to avoid potential duplicates that can result from random generation or multiple users generating same id